src/template/ar/careers.ejs

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="ar" translate="no">
+
+<head>
+    <title>وظائف | Martspec</title>
+    <meta name="description"
+        content="فريقنا عبارة عن مجموعة من المهنيين الشباب الواعدين الذين يستمتعون بأداء عملهم. نحن نبحث عن الأشخاص الموهوبين والطموحين والمستعدين للنمو والتطور معنا.">
+
```

Comment:
Please add to all templates canonical link that should be copy of URL in the x-default element

`    <link rel="canonical" href="https://martspec.com/ar/careers">`